internal/cmd/request.go

```diff
@@ -115,3 +123,71 @@ func PrintCMDResponseFull(request any, response core.APIResponse, responseType a
 		fmt.Println(util.MustToJSONIndent(responseContent))
 	}
 }
+
+// Attempt to convert an API response to a TSV table.
+// Returns false if there are issues or the TSV table if it was successful.
+func PrintCMDResponseTable(response core.APIResponse) (string, bool) {
+	responseContent, ok := response.Content.(map[string]any)
+	if !ok {
+		return "", false
+	}
+
+	// Don't try to format a response that has multiple keys.
+	if len(responseContent) != 1 {
+		return "", false
+	}
+
+	responseContentKey := reflect.ValueOf(responseContent).MapKeys()
```

Comment:
Why not just range over the values (which you know there is only 1).

Reflection is super cool, but we should only do it when we need it.